src/PDPVerifier.sol

```diff
@@ -321,11 +329,16 @@ contract PDPVerifier is Initializable, UUPSUpgradeable, OwnableUpgradeable {
             addOneRoot(setId, i, rootData[i].root, rootData[i].rawSize);
         }
 
+        uint256[] memory rootIds = new uint256[](rootData.length);
+        for (uint256 i = 0; i < rootData.length; i++) {
+            rootIds[i] = firstAdded + i;
```

Comment:
Any good reason not to do this inside the other loop?

Reply:
moved this